src/applications/vaos/tests/project-cheetah/components/ReviewPage.unit.spec.jsx

```diff
@@ -105,7 +105,7 @@ describe('VAOS vaccine flow <ReviewPage>', () => {
     });
 
     await screen.findByText(/COVID-19 vaccine/i);
-    const [pageHeading, descHeading, clinicHeading] = screen.getAllByRole(
+    const [pageHeading, descHeading, facilityHeading] = screen.getAllByRole(
```

Comment:
Interesting. Is this order dependent?

Reply:
Yeah, this is just pulling the first three headers on the page. The copy changes flipped it so that the facility name is now the header, and the clinic name is regular text. So I renamed the variable here to be more accurate.